server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -859,19 +857,14 @@ private void mergeMetadataRecords(MergeInfo info) throws AccumuloException {
     KeyExtent previousKeyExtent = null;
     KeyExtent lastExtent = null;
 
-    try (BatchWriter bw = client.createBatchWriter(targetSystemTable)) {
-      // Check if we have already previously fenced the tablets
-      if (highTablet.isMerged()) {
-        Manager.log.debug("tablet metadata already fenced for merge");
-        if (!highTablet.hasPrevRowColumn()) {
-          Manager.log.debug("tablet already merged, returning");
-          return;
-        }
-        // Continue and delete tablets as we already finished fencing
-        deleteTablets(info, scanRange, bw, client);
-        return;
-      }
+    // Check if we have already previously fenced the tablets
+    if (highTablet.isMerged()) {
+      Manager.log.debug("tablet metadata already fenced for merge");
```

Comment:
Would be good to include some info in the log message that identifies specific merge operation